Hudl.Mjolnir/Command/CommandContext.cs

```diff
@@ -132,18 +143,18 @@ internal static ICircuitBreaker GetCircuitBreaker(GroupKey key)
                     new ConfigurableValue<bool>("mjolnir.breaker." + key + ".forceTripped", DefaultBreakerForceTripped),
                     new ConfigurableValue<bool>("mjolnir.breaker." + key + ".forceFixed", DefaultBreakerForceFixed));
 
-                return new FailurePercentageCircuitBreaker(key, metrics, Stats, properties);
+                return new FailurePercentageCircuitBreaker(key, metrics, Stats, MetricEvents, properties);
             });
         }
 
-        private static ICommandMetrics GetCommandMetrics(GroupKey key)
+        private ICommandMetrics GetCommandMetrics(GroupKey key)
         {
             if (key == null)
             {
                 throw new ArgumentNullException("key");
             }
 
-            return Instance._metrics.GetOrAddSafe(key, k =>
+            return _metrics.GetOrAddSafe(key, k =>
```

Comment:
Same again. In fact since one calls the other, there's a very very slim chance that they become out of sync. :thought_balloon: